src/request/request.builder.js

```diff
@@ -176,12 +220,18 @@ class RequestBuilder {
     // Build endpoint and url.
     const endpointPath = `${this.endpoint}?${queryString}`;
 
+    const requestCanSendPayload = yotiCommon.requestCanSendPayload(this.method);
```

Comment:
Just a bit weird to reuse the same name as the method for that const, isn't `canSendPayload` good enough?